db/schema.rb

```diff
@@ -24,15 +24,22 @@
     t.index ["recipe_id"], name: "index_comments_on_recipe_id"
   end
 
+  create_table "rates", force: :cascade do |t|
+    t.bigint "recipe_id", null: false
+    t.integer "vote", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["recipe_id"], name: "index_rates_on_recipe_id"
+  end
+
   create_table "recipes", force: :cascade do |t|
     t.string "name", null: false
     t.text "content"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "video_link"
-    t.integer "cooking_time"
-    t.decimal "price"
```

Comment:
Why price is removed?

Reply:
It shouldn't be there. Price field is still in PR and it isn't in any migration.